static/app/components/quickTrace/index.tsx

```diff
@@ -86,8 +88,52 @@ export default function QuickTrace({
   const traceLength = quickTrace.trace && quickTrace.trace.length;
   const {root, ancestors, parent, children, descendants, current} = parsedQuickTrace;
 
+  const hasReplay =
+    Boolean(event.entries?.find(({type}) => type === 'breadcrumbs')) &&
+    Boolean(event.tags?.find(({key}) => key === 'replayId')?.value);
+
+  // Using click handler to navigate to breadcrumbs/replay section because EventNodeReplay (Tag)
+  // treats href as an external link and the `to` prop doesn't initiate a scroll to the section.
+  const handleReplayClick = () => {
+    if (hasReplay) {
+      document.getElementById('breadcrumbs')?.scrollIntoView();
+    }
+  };
+
   const nodes: React.ReactNode[] = [];
 
+  if (organization.features.includes('session-replay-ui')) {
+    nodes.push(
+      <Projects
+        key="replay-projects"
+        orgId={organization.slug}
+        slugs={[current.project_slug]}
+      >
+        {({projects}) => {
+          const project = projects.find(p => p.slug === current.project_slug);
+          if (projectSupportsReplay(project)) {
+            return (
+              <EventNodeReplay
+                data-test-id="replay-node"
+                to={{
+                  ...location,
+                  hash: '#breadcrumbs',
+                }}
+                onClick={handleReplayClick}
```

Comment:
Would also be fine to do `onClick={hasReplay ? handleReplayClick : undefined }` here. Nice symmetry with the other props. Then the `handleReplayClick` handling could be inline